gwtbootstrap3/src/main/java/org/gwtbootstrap3/client/ui/base/form/AbstractForm.java

```diff
@@ -455,12 +455,12 @@ private void createFrame() {
     }
 
     /**
-     * Fire a {@link FormPanel.SubmitEvent}.
+     * Fire a {@link AbstractForm.SubmitEvent}.
      *
      * @return true to continue, false if canceled
      */
     private boolean fireSubmitEvent() {
-        FormPanel.SubmitEvent event = new FormPanel.SubmitEvent();
+        SubmitEvent event = new SubmitEvent();
```

Comment:
Hey,

In this case FormPanel's SubmitEvent is not used, as AbstractForm has its own SubmitEvent and SubmitCompleteEvent classes which are copies of the ones from GWT's FormPanel (the only difference is that one field in the SubmitCompleteEvent is not declared final). I think removing the duplicate classes and using FormPanel ones in AbstractForm would be a cleaner choice, but I was not aware of the author's intentions. If the copy-paste was not intended, then I can gladly make a cleanup.